### DISCUSSION TITLE
Is it possible to set a specific name for the snapshot?

### DISCUSSION THREAD

**Discussion by psayan** [backube/snapscheduler#379] [General]
I want to retain only one snapshot of each PVC, is there a way to name the snapshot in a certain fashion so that I can identify the PVC backup from the name only?

**Comment by psayan** [comment]
For example, let's say I do not want to include the <schedule_name>-<snapshot_time>. Is it possible?

**Comment by JohnStrunk** [comment]
That's not possible today. An implementation of #113 may solve your use-case though.
